pcdsdevices/liquid_jet.py

```diff
@@ -0,0 +1,30 @@
+from ophyd import Component as Cpt
+
+from .device import GroupDevice
+from .epics_motor import BeckhoffAxis
+from .interface import BaseInterface
+
+
+class LiquidJet(BaseInterface, GroupDevice):
+    """
+    Liquid jet motion class.
+
+    This class controls the axis for the liquid jet setups
+    """
+    tab_component_names = True
+    #_icon = 'fa.minus-square'
```

Comment:
```suggestion
    # _icon = 'fa.minus-square'
```
Style dictates we put a space after the comment '#'.  Did you want this to have an icon though?